Adding new PS cmdlets (#11249)
MicrosoftDocs/office-docs-powershell: teams/teams-ps/teams/Grant-CsTeamsSharedCallingRoutingPolicy.md

applicable: Microsoft Teams
author: jenstrier
ms.author: jenstr
ms.reviewer: 
manager:
schema: 2.0.0
---

# Grant-CsTeamsSharedCallingRoutingPolicy

## SYNOPSIS

Assigns a specific Teams shared calling routing policy to a user, a group of users, or sets the Global policy instance.

## SYNTAX

### Identity (Default)
```
Grant-CsTeamsSharedCallingRoutingPolicy [<CommonParameters>]
```

### GrantToUser
```
Grant-CsTeamsSharedCallingRoutingPolicy [[-PolicyName] <string>] -Identity <string> [<CommonParameters>]
```

### GrantToTenant
```
Grant-CsTeamsSharedCallingRoutingPolicy -Global [[-PolicyName] <string>] [-Force] [<CommonParameters>]
```

### GrantToGroup
```
Grant-CsTeamsSharedCallingRoutingPolicy [-Group] <string> [[-PolicyName] <string>] -Rank <int> [<CommonParameters>]
```

## EXAMPLES

### EXAMPLE 1
```
Grant-CsTeamsSharedCallingRoutingPolicy -Identity "user@contoso.com" -PolicyName "Seattle"
```
The command shown in Example 1 assigns the per-user Teams shared calling routing policy instance Seattle to the user user@contoso.com.

### EXAMPLE 2
```
Grant-CsTeamsSharedCallingRoutingPolicy -PolicyName "Seattle" -Global
```

> Example 2 assigns the per-user Teams shared calling routing policy instance Seattle to all the users in the tenant, except any that have an explicit Teams shared calling routing policy assignment.

Reviewer: ```suggestion
Example 2 assigns the per-user Teams shared calling routing policy instance Seattle to all the users in the organization, except any that have an explicit Teams shared calling routing policy assignment.
```

## PARAMETERS

### -Identity

> Indicates the Identity of the user account to be assigned the per-user Teams shared calling routing policy. User Identities can be specified using one of the following formats: the user's SIP address, the user's user principal name (UPN), or the user's ObjectId or Identity.

Reviewer: ```suggestion
Indicates the Identity of the user account to be assigned the per-user Teams shared calling routing policy. User identities can be specified using one of the following formats: the user's SIP address, the user's user principal name (UPN), or the user's ObjectId or Identity.
```

```yaml
Type: String
Parameter Sets: Identity
Aliases:

Required: True
Position: Named
Default value: None
Accept pipeline input: True (ByPropertyName, ByValue)
Accept wildcard characters: False
```

### -PolicyName

> "Name" of the policy to be assigned. The PolicyName is simply the policy Identity minus the policy scope (the "tag:" prefix). For example, a policy with the Identity tag:Seattle has a PolicyName equal to Seattle.

Reviewer: ```suggestion
Name of the policy to be assigned. The PolicyName is simply the policy Identity minus the policy scope (the "tag:" prefix). For example, a policy with the Identity tag:Seattle has a PolicyName equal to Seattle.
```

To unassign a per-user policy previously assigned to a user, set the PolicyName to a null value ($Null).

```yaml
Type: String
Parameter Sets: (All)
Aliases:

Required: False
Position: Named
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

### -Global

> When you use this cmdlet without specifying a user identity, the policy applies to all users in your tenant, except any that have an explicit policy assignment. To skip a warning when you do this operation, specify this parameter.

Reviewer: ```suggestion
When you use this cmdlet without specifying a user identity, the policy applies to all users in your organization, except any that have an explicit policy assignment. To prevent a warning when you carry out this operation, specify this parameter.
```
